src/test_scenes/sceneScrollBar.js

```diff
@@ -1,26 +1,26 @@
 import React from 'react';
 import { MathUtils } from '../utils/mathUtils';
-import { LinearLayout, Platform, ScrollBar, View } from 'magic-script-components';
+import { AnchorPoint, LinearLayout, Orientation, Platform, ScrollBar, View } from 'magic-script-components';
 
 class SceneScrollBar extends React.Component {
   render() {
     const quat = (Platform.OS == 'Lumin') ? MathUtils.rotateBy(1.5707963268, [0, 0, -1]) : [0,0,0,0];
     return (
       <View position={this.props.position}>
         <LinearLayout
-          anchorPoint={'center-center'}
+          anchorPoint={AnchorPoint.centerCenter}
           defaultItemPadding={[0.07, 0, 0.07, 0]}
-          orientation={'horizontal'}>
+          orientation={Orientation.horizontal}>
           <ScrollBar thumbSize={0.25} thumbPosition={0.33} length={2.0} thickness={0.04} rotation={quat} />
           <LinearLayout>
-            <LinearLayout defaultItemPadding={[0.07, 0.07, 0.07, 0.07]} orientation={'horizontal'}>
+            <LinearLayout defaultItemPadding={[0.07, 0.07, 0.07, 0.07]} orientation={Orientation.horizontal}>
               <ScrollBar thumbPosition={0.1} length={1.2} thickness={0.07} rotation={quat} />
-              <ScrollBar thumbSize={7.0} thumbPosition={0.33} length={0.7} thickness={0.11} rotation={quat} />
+              <ScrollBar thumbSize={1.0} thumbPosition={0.33} length={0.7} thickness={0.11} rotation={quat} />
```

Comment:
why was this 7 before?

Reply:
I don't know why there was 7 before, but only values between 0 and 1 are valid. 0 means a minimum size of the thumb (a dot), 1 means a maximum value (full length of the scrollbar). I set PropType which validates if the `thumbSize` value is within a [0,1] range.